cli/templatecreate.go

```diff
@@ -49,6 +49,10 @@ func templateCreate() *cobra.Command {
 				templateName = args[0]
 			}
 
+			if len(templateName) > 31 {
```

Comment:
`len` counts the number of bytes,5 instead we can use [`utf8.RuneCountInString`](https://pkg.go.dev/unicode/utf8#RuneCountInString) to count the number of actual characters!

Reply:
Oops, TIL! I'll fix that. Thanks for the tip!